content/ja/docs/concepts/scheduling-eviction/pod-priority-preemption.md

```diff
@@ -51,7 +51,8 @@ PriorityClassオブジェクトのメタデータの`name`フィールドにて
 値が大きいほど、高い優先度を示します。
 PriorityClassオブジェクトの名称は[DNSサブドメイン名](/ja/docs/concepts/overview/working-with-objects/names#dns-subdomain-names)として適切であり、かつ`system-`から始まってはいけません。
 
-PriorityClassオブジェクトは10億以下の任意の32ビットの整数値を持つことができます。
+
+PriorityClass オブジェクトには、 1000000000 以下の任意の 32 ビット整数値を含めることができます。これは、PriorityClass オブジェクトの値の範囲が -2147483648 から 1000000000 までであることを意味します。
```

Comment:
```suggestion
PriorityClassオブジェクトは10億以下の任意の32ビットの整数値を持つことができます。これは、PriorityClassオブジェクトの値の範囲が-2147483648から1000000000までであることを意味します。
```
- The added blank line is not needed.
- The first sentence need not be changed I think.
- Japanese pages do not include spaces between Japanese and alphanumeric words.

Reply:
Thanks for reviewing @t-inu